### DISCUSSION TITLE
Fortnite mobile version

### DISCUSSION THREAD

**Discussion by EnriqueAlan** [redphx/better-xcloud#256] [General]
The feature to play the desktop version of fortnite on mobile is great! Is it possible to have to opposite added as well? Being able to play the mobile version on desktop?

This could be useful for starting up bot lobbies, which require a mobile session, though I understand if this a very niche use and not worth the effort of adding it

**Comment by DmonUwU** [comment]
I would appreciate it if you could implement an option to select the mobile version of Fortnite, allowing users to utilize mobile features such as auto-fire and point-and-shoot mode directly from the browser. Despite attempting to force this mode by altering the user agent, I encountered difficulties, and the desired functionality was not achieved.
